thirdparty/jquery-entwine/jquery.entwine.js

```diff
@@ -2295,28 +2295,3 @@ Sizzle is good for finding elements for a selector, but not so good for telling
 	});
 	
 })(jQuery);
-
-
-/* src/jquery.entwine.legacy.js */
-
-(function($) {	
-	
-	// Adds back concrete methods for backwards compatibility
-	$.concrete = $.entwine;
-	$.fn.concrete = $.fn.entwine;
-	$.fn.concreteData = $.fn.entwineData;
-	
-	// Use addHandler to hack in the namespace.$.concrete equivilent to the namespace.$.entwine namespace-injection
-	$.entwine.Namespace.addHandler({
-		order: 100,
-		on: function(selector, k, v) { return false; },
-	
-		namespaceMethodOverrides: function(namespace){
-			namespace.$.concrete = namespace.$.entwine;
-			namespace.injectee.concrete = namespace.injectee.entwine;
-			namespace.injectee.concreteData = namespace.injectee.entwineData;
-			return {};
-		}
-	});
-
-})(jQuery);
```

Comment:
`concrete` was a very very very very very old way of invoking entwine. There's virtually 0 chance anyone is using it.